src/coupling.jl

```diff
@@ -158,17 +158,20 @@ function build_split_jumps(prob::DiffEqBase.AbstractJumpProblem,
         push!(jumps, ConstantRateJump(new_rate, new_affect!))
         # only prob
         new_affect! = affect!
-        new_rate = (u, p, t) -> rate(u.u, p, t) -
-                                min(rate(u.u, p, t), rate_control(u.u_control, p, t))
+        new_rate = (u, p,
+            t) -> rate(u.u, p, t) -
+                  min(rate(u.u, p, t), rate_control(u.u_control, p, t))
         push!(jumps, ConstantRateJump(new_rate, new_affect!))
         # only prob_control
         new_affect! = function (integrator)
             flip_u!(integrator.u)
             affect!(integrator)
             flip_u!(integrator.u)
         end
-        new_rate = (u, p, t) -> rate_control(u.u_control, p, t) -
-                                min(rate(u.u, p, t), rate_control(u.u_control, p, t))
+        new_rate = (u,
+            p,
+            t) -> rate_control(u.u_control, p, t) -
+                  min(rate(u.u, p, t), rate_control(u.u_control, p, t))
```

Comment:
So Juliaformatter is still as broken as it was over a year ago when we stopped using it? Let's please revert this one too...